src/deploy/functions/release/fabricator.ts

```diff
@@ -248,6 +250,13 @@ export class Fabricator {
           })
           .catch(rethrowAs(endpoint, "set invoker"));
       }
+    } else if (backend.isBlockingTriggered(endpoint)) {
+      // Blocking functions should always be public
+      await this.executor
+        .run(async () => {
+          await gcf.setInvokerCreate(endpoint.project, backend.functionName(endpoint), ["public"]);
```

Comment:
Here and elsewhere, it's `public` because it's an auth blocking function, not because it's a blocking function. Please make the code forwards compatible to other blocking trigger types.

Reply:
Added a check
```
if (backend.isBlockingTriggered(endpoint) && AUTH_BLOCKING_EVENTS.includes(endpoint.blockingTrigger.eventType)) { ... }
```